common/services/prismic/parsers.js

```diff
@@ -454,6 +456,40 @@ export function parseSingleLevelGroup(
   /* eslint-enable */
 }
 
+export function parseFormat(frag: Object): ?Format {
+  return isDocumentLink(frag)
+    ? {
+        id: frag.id,
+        title: parseTitle(frag.data.title),
+        description: asHtml(frag.data.description),
+      }
+    : null;
+}
+
+function parseLink(url): ?string {
+  if (url) {
+    if (url.link_type === 'Web' || url.link_type === 'Media') {
+      return url.url;
+    } else if (url.link_type === 'Document' && isDocumentLink(url)) {
+      return `/${url.type}/${url.id}`;
```

Comment:
I think we can use the [linkResolvers](https://prismic.io/docs/javascript/beyond-the-api/link-resolving) for this?

https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/services/prismic/link-resolver.js